requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -0,0 +1,151 @@
+# Request for comments: Onboarding emails
+
+## Context
+
+We had previously tracked what the email onboarding flow looked like as a series of numbered issues, progressing up to 3.3/3.4. However, this made it difficult for people who don't have access to Customer.io to get a sense of what the onboarding flows looked like - especially because they spread across multiple flows. It also meant the flow was a bit of a hodge-podge patchwork of ideas. 
+
+So, now we have this file. Whenever a change is introduced, we PR and update to this file. In that way, this file always describes what the onboarding flow looks like, changes are easy, and in-line comments are possible. 
```

Comment:
Should we just have this in handbook somewhere and PR to it? Don't feel strongly if this is easier.